Makefile.mk

```diff
@@ -62,6 +62,18 @@ else
 	./api_build/api
 endif
 
+run-ballot-parser: build
+	if [ ! -d "ballot_parser_build" ]; then mkdir ballot_parser_build; fi
+ifeq ($(OPERATING_SYSTEM),Windows)
+	CMAKE_PREFIX_PATH="./build/ElectionGuard" cmake -S examples/ballot_parser -B ballot_parser_build -G "MSYS Makefiles"
+	cmake --build ballot_parser_build --target ballot_parser
+	PATH=$(PWD)/build:$$PATH; ./ballot_parser_build/ballot_parser
+else
+	ElectionGuard_DIR=$(ELECTIONGUARD_DIR) cmake -S examples/ballot_parser -B ballot_parser_build
+	cmake --build ballot_parser_build --target ballot_parser
+	./ballot_parser_build/ballot_parser
```

Comment:
maybe pass in the default `./examples/ballot_parser/ballot_samples` path here?

Reply:
Fixed